### ISSUE TITLE
你好，你是？

### ISSUE THREAD

**Issue by yamaidie** [ncxiaomai/qingtian#1]
你也是师大的吗？交个朋友

**Comment by ncxiaomai** [comment]
是的 发自网易邮箱大师 在2016年07月15日 23:20 ，yamaidie写道： 你也是师大的吗？交个朋友 — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by ncxiaomai** [comment]
我是陈声钰 发自网易邮箱大师 在2016年07月15日 23:20 ，yamaidie写道： 你也是师大的吗？交个朋友 — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or mute the thread.
